ExadelTimeTrackingSystem/ExadelTimeTrackingSystem.BusinessLogic/Mappings/TaskProfile.cs

```diff
@@ -0,0 +1,19 @@
+﻿namespace ExadelTimeTrackingSystem.BusinessLogic.Mappings
+{
+    using System;
+    using System.Threading.Tasks;
+    using AutoMapper;
+    using ExadelTimeTrackingSystem.BusinessLogic.DTOs;
+
+    public class TaskProfile : Profile
+    {
+        public TaskProfile()
+        {
+            CreateMap<Task, TaskDTO>();
+            CreateMap<CreateTaskDTO, Task>()
+                .ForMember(dest => dest.Status, opt => opt.MapFrom(src => src.Status));
```

Comment:
don't need this line. instead of it CreateMap from Status model to Status DTO and vice versa using ReverseMap